src/Components/HomePage.tsx

```diff
@@ -0,0 +1,94 @@
+import React, { FC, useState } from 'react';
+import {
+    Grid,
+    List,
+    Snackbar
+} from '@material-ui/core';
+import MuiAlert from '@material-ui/core/Alert';
+import Task from './Task';
+import TaskForm from './TaskForm';
+import { ITask } from '../interfaces/task';
+
+const TWO_SECONDS_AND_HALF = 2500;
+
+const HomePage: FC = () => {
+    const [tasks, setTasks] = useState([{
+        title: 'Tarefa 1',
+        id: Math.random(),
+        done: false
+    }]);
+
+    const [infoMessageModal, setInfoMessageModal] = useState('');
+
+    const taskAddedHandler = (values: ITask) => {
+        setTasks((prevState) => [
+            {
+                title: values.title,
+                id: Math.random(),
+                done: false
+            },
+            ...prevState
+        ]);
+        setInfoMessageModal('Tarefa adicionada com sucesso.');
+    };
+
+    const removeTaskHandler = (id: number) => {
+        const newTasks = tasks.filter((task) => task.id !== id);
+        setTasks(newTasks);
+        setInfoMessageModal('Tarefa removida com sucesso.');
+    };
+
+    const closeModalHandler = (_: any, reason: string) => {
+        const isClickIngAway = reason === 'clickaway';
+        if (isClickIngAway) {
+            return;
+        };
+        setInfoMessageModal('');
+    };
+
+    const toggleCheckedTaskProp = (id: number) => {
+        const taskIndex = tasks.findIndex((currentTask) => currentTask.id === id);
+        tasks[taskIndex].done = !tasks[taskIndex].done;
+        setTasks([...tasks]);
+    };
+
+    return (
+        <Grid
+            container
+            sx={{
+                height: '100%',
+                width: '50vw',
+                pt: 10,
+                margin: '0 auto',
```

Comment:
```suggestion
                height: '100%',
                pt: 10,
                width: '50vw',
```

To center components use `Stack` instead of grid.

Reply:
Resolved on 0fb339cfe72ba3858ea75d18d4c681c6926971b4